src/Microsoft.TestPlatform.CommunicationUtilities/TestRequestSender.cs

```diff
@@ -622,8 +642,24 @@ private void OnTestRunAbort(IInternalTestRunEventsHandler testRunEventsHandler,
         SetOperationComplete();
 
         var reason = GetAbortErrorMessage(exception, getClientError);
```

Comment:
This could probably return a tuple, or out result with a boolean to indicate if we got default result or not. Rather than checking the literal text.